scripts.js

```diff
@@ -1,175 +0,0 @@
-//These functions prepares the game for the player, starting at "prepareTheGame()"
```

Comment:
`// These functions prepares the game for the player, starting at "prepareTheGame()"`